### PR TITLE
No results

### PR DESCRIPTION
**Pull request by alexisgrow**

### CODE REVIEW

**Review by tmeyer2115 — COMMENTED** [review]
Is the failing test due to the rollup includes stuff? If so, I can just lg!

**Review by tmeyer2115 — APPROVED** [review]

File: src/ui/templates/icons/icon.hbs [hunk 1]
```diff
@@ -2,6 +2,7 @@
   {{#if iconUrl}}
     <img src="{{iconUrl}}" alt="" class="Icon-image">
   {{else}}
+    {{iconName}}
```
Comment (tmeyer2115) [inline src/ui/templates/icons/icon.hbs]: Is this printing the string iconName before the actual icon?

File: src/ui/components/icons/iconcomponent.js [hunk 2]
```diff
@@ -62,7 +62,6 @@ export default class IconComponent extends Component {
     return super.setState(Object.assign(data, {
       iconUrl: this.iconUrl,
       iconName: this.iconName,
-      name: this.iconName ? this.iconName : 'custom',
```
Comment (tmeyer2115) [inline src/ui/components/icons/iconcomponent.js]: If we remove this, does the {{name}} reference in icon.hbs also need to be removed?

File: src/ui/templates/icons/icon.hbs [hunk 3]
```diff
@@ -2,7 +2,6 @@
   {{#if iconUrl}}
     <img src="{{iconUrl}}" alt="" class="Icon-image">
   {{else}}
-    {{iconName}}
```
Comment (tmeyer2115) [inline src/ui/templates/icons/icon.hbs]: Whoops, was looking at the commits one by one, ignore my comment about this

File: src/ui/components/icons/iconcomponent.js (hunk 2 above)
Comment (alexisgrow) [reply src/ui/components/icons/iconcomponent.js]: Ahh, you're right, this one should not be removed!

**Pull request merged**